debian/alignak-all.postinst

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+# postinst script for alignak-common
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    configure)
+        ## Create user and group
+        # Note: if the user exists, it's properties won't be changed (gid, home, shell)
+        adduser  --quiet --system --home /var/lib/alignak --no-create-home --group alignak || true
+
+        ## Create nagios group
+        addgroup --system nagios || true
+
+        ## Add alignak to nagios group
+        id -Gn alignak |grep -E '(^|[[:blank:]])nagios($|[[:blank:]])' >/dev/null ||
+            adduser alignak nagios
+
+        ## Create directories with proper permissions
+        for i in /var/run/alignak /var/log/alignak /var/lib/alignak
+        do
+            mkdir -p $i
+            chown -R alignak:alignak $i
+            # only do something when no setting exists
+            if ! dpkg-statoverride --list $i >/dev/null 2>&1
+            then
+                dpkg-statoverride --update --add alignak alignak 02750 $i
+            fi
+        done
+
+        ## Ensure alignak starts automatically
+        update-rc.d -f alignak-arbiter defaults > /dev/null
+        update-rc.d -f alignak-scheduler defaults > /dev/null
+        update-rc.d -f alignak-poller defaults > /dev/null
+        update-rc.d -f alignak-broker defaults > /dev/null
+        update-rc.d -f alignak-reactionner defaults > /dev/null
+        update-rc.d -f alignak-receiver defaults > /dev/null
+        update-rc.d -f alignak defaults > /dev/null
+
+        # Move old config files
```

Comment:
This part seems useless: We don't have any older alignak deb packaging version ...